### PR TITLE
Fix and remove some broken links 

### PR DESCRIPTION
**Pull request by teamwolfyta**
Hello, I noticed some links weren't working so went through and (hopefully) fixed them, There was also a warning that contained a broken link but I couldn't figure out what kind of content that link was linking too and the warning message didn't seem too relevant so I removed the warning all together.

### CODE REVIEW

File: src/content/docs/commands/prefix-commands.mdx [hunk 1]
```diff
@@ -66,7 +66,7 @@ Seyfert supports subcommands within prefix commands.
 
 ## Replying to prefix commands
 
-By default when you use `write` or `editOrReply` function it will only send a message to the channel where the command was fired. To make the command being replied you have to set `reply` suboption inside `command` option in the client options. This option is a callback which first parameter is the [`CommandContext`](/api/classes/commandcontext) and expects a boolean to be returned, if is `true` it will reply to the command, if not it will just send a message. 
+By default when you use `write` or `editOrReply` function it will only send a message to the channel where the command was fired. To make the command being replied you have to set `reply` suboption inside `command` option in the client options. This option is a callback which first parameter is the [`CommandContext`](/api/classes/commandcontext) and expects a boolean to be returned, if is `true` it will reply to the command, if not it will just send a message.
```
Comment (socram03) [inline src/content/docs/commands/prefix-commands.mdx]: CommandContext url?

**Review by socram03 — DISMISSED** [review]

**Review by EvilG-MC — DISMISSED** [review]

**Comment by teamwolfyta** [comment]
Turns out fixing links is kind of difficult and annoying... I was getting lost in all the text so employed a different method of finding broken links, In trying that other method I found a bunch of issues so went through and (hopefully) fixed them.

Break down of what I did: 
- Use `/commands/options` instead of `./options`.
- Use `#usando-opciones` instead of `#using-options` so it actually takes the viewer to the correct header.
- Replace links that no longer exist with new links.
- Remove any messages requiring links that don't exist anymore.

**Review by socram03 — APPROVED** [review]

**Review by EvilG-MC — APPROVED** [review]
🐐💪

**Comment by socram03** [comment]
Amazing work, I love you so much 
alarm

**Pull request merged**